Screensharing

negzi opened the pull request:
Example for screen sharing.

Commit:
Enable screensharing in chrome

Commit:
Use video element instead

Commit:
Add chrome extension

stefanalund reviewed (changes requested):
I think you should add a README.md file in the `screen-sharing` folder and describe how to create the extension from this "template". Include info on how to submit to Google Chrome Web Store.

stefanalund commented on screen-sharing/screenSharing.html:
```diff
@@ -0,0 +1,115 @@
+        </div>
+</div>
+<script type="application/javascript">
+var video_status = false;
```
This is not a very good variable name 😏

negzi replied:
hehe thats python :)

Commit:
Add readme for chrome extension.
Do some fixes:
    - change a video_status variable to videoStatus
    - refactor stopSCreen function

stefanalund reviewed (approved):
LGTM

Commit:
Modify main readme and index files in c3-web-examples
